robocorp-code/package.json

```diff
@@ -955,18 +955,22 @@
         "prettier-fix": "npx prettier -w vscode-client/**/*.ts"
     },
     "devDependencies": {
+        "@types/adm-zip": "^0.5.0",
         "@types/mocha": "^2.2.32",
         "@types/node": "^13.0.00",
+        "@types/rimraf": "^3.0.2",
         "@types/vscode": "1.65.0",
         "prettier": "2.4.1",
-        "vscode-test": "1.5.1",
-        "typescript": "^4.5.4"
+        "typescript": "^4.5.4",
+        "vscode-test": "1.5.1"
     },
     "dependencies": {
-        "vscode-languageclient": "^7.0.0",
-        "path-exists": "^4.0.0",
+        "adm-zip": "^0.5.9",
         "http-proxy-agent": "^2.1.0",
         "https-proxy-agent": "^2.2.4",
+        "path-exists": "^4.0.0",
```

Comment:
The package.json can't be changed directly. Rather, edit `robocorp-code/codegen/codegen_package.py` to add/change dependencies (and then rerun that script in python).